daprdocs/content/en/reference/components-reference/supported-bindings/redis.md

```diff
@@ -88,6 +88,54 @@ You can store a record in Redis using the `create` operation. This sets a key to
 
 An HTTP 204 (No Content) and empty body is returned if successful.
 
+- `get`
```

Comment:
Let's model this section similarly to the [postgresql component](https://docs.dapr.io/reference/components-reference/supported-bindings/postgres/#binding-support)... so from lines 68 - 91:

```
- `create`
- `get`
- `delete`

### create

You can store a record in Redis using the `create` operation. This sets a key to hold a value. If the key already exists, the value is overwritten.

#### Request

```json
{
  "operation": "create",
  "metadata": {
    "key": "key1"
  },
  "data": {
    "Hello": "World",
    "Lorem": "Ipsum"
  }
}
```
```
#### Response

An HTTP 204 (No Content) and empty body is returned if successful.

### get
```